_docs/deployment/managed-services.md

```diff
@@ -35,9 +35,17 @@ Create a new service instance by specifying a service, plan, and a name of your
 % cf create-service SERVICE_NAME PLAN_NAME INSTANCE_NAME
 ```
 
+### Sharing service instances
+
+Sharing of service instances allows customers to share service instances between spaces inside their organization as a way to consolidate service instance usage and avoid data duplication in their service instances.  Customers need to be aware that sharing service instances between spaces has limitations and can expose data in those service instances across environment boundaries.  Please see [Sharing Service Instances](https://docs.cloudfoundry.org/devguide/services/sharing-instances.html) for more information on limitations and security concerns.
```

Comment:
Can we say "users" instead of "customers"?

Reply:
Sure i'm fine with that @bengerman13